elide-core/src/main/java/com/yahoo/elide/core/filter/expression/AndFilterExpression.java

```diff
@@ -17,6 +17,31 @@
     @Getter private FilterExpression left;
     @Getter private FilterExpression right;
 
+    /**
+     * Returns a new {@link AndFilterExpression} instance with the specified null-able left and right operands.
+     * <p>
+     * If both left and right are not {@link null}, this method produces the same instance as
+     * {@link #AndFilterExpression(FilterExpression, FilterExpression)} does. If only one of them is {@link null}, the
+     * other non-null is returned with no modification. If both left and right are {@link null}, this method returns
+     * {@code null}.
+     *
+     * @param left  The provided left {@link FilterExpression}
+     * @param right  The provided right {@link FilterExpression}
+     *
+     * @return a new {@link AndFilterExpression} instance or {@code null}
+     */
+    public static FilterExpression withLeftAndRight(FilterExpression left, FilterExpression right) {
+        if (left != null && right != null) {
+            return new AndFilterExpression(left, right);
+        } else if (left == null && right == null) {
```

Comment:
This can be shortened like this:
```java
        if (left != null && right != null) {
            return new AndFilterExpression(left, right);
        } else if (left == null) {
             return right;
        } 
        return left;
```